docker-compose.dev.yml

```diff
@@ -100,6 +107,11 @@ services:
 networks:
   redwood:
     driver: bridge
+  ls:
+    ipam:
+      config:
+        # Specify the subnet range for IP address allocation
+        - subnet: 10.0.2.0/24
```

Comment:
Needed for proper SQS message routing in the local development environment: https://github.com/localstack/localstack/issues/10590#issuecomment-2177299093